ape_safe/accounts.py

```diff
@@ -42,9 +44,22 @@
 class SafeContainer(AccountContainerAPI):
     _accounts: dict[str, "SafeAccount"] = {}
 
+    @property
+    def config(self) -> SafeConfig:
+        return cast(SafeConfig, self.config_manager["safe"])
+
     @property
     def _account_files(self) -> Iterator[Path]:
-        yield from self.data_folder.glob("*.json")
+        account_files = list(self.data_folder.glob("*.json"))
+
+        # NOTE: Make sure these Safes exist in our local cache
+        for required_safe, safe_cache_data in self.config.require.items():
```

Comment:
Does this handle duplicate safes? That may happen if I put 1 in my project's config and 1 in my global and they happen to be the same.

Reply:
`if required_safe not in map(lambda p: p.stem, account_files)` basically says "If we don't have this alias, add it"

I will write a note about that

Reply:
ab29541